README.md

```diff
@@ -121,9 +121,9 @@ The output is an object with these top-level properties:
 Each _changed resource_ has the following properties:
 
 - **`action`:** One of `"create"`, `"destroy"`, `"replace"`, `"update"`
-- **`module`:** Name of a module that contains a resource (only present when module is not `root`)
```

Comment:
Deleted since this is a duplicate description, and the other description is better IMO.